bugnote_view_inc.php

```diff
@@ -216,20 +220,36 @@
 
 				# show edit button if the user is allowed to edit this bugnote
 				if( $t_can_edit_bugnote ) {
-					print_button( 'bugnote_edit_page.php?bugnote_id='.$t_bugnote->id, lang_get( 'bugnote_edit_link' ) );
+					print_button(
+						'bugnote_edit_page.php',
+						lang_get( 'bugnote_edit_link' ),
+						array( 'bugnote_id' => $t_bugnote->id ),
+						OFF );
```

Comment:
It is OFF because "bugnote_edit_page.php" does not check for token
maybe we should check for it there too.

In any way, i agree on commenting about the OFF in case it stays